### PR TITLE
✨ Brid player support for child JSON config

### PR DESCRIPTION
**Pull request by grajzer**
Support for providing additional config using child JSON.

### CODE REVIEW

**Comment by erwinmombay** [comment]
@grajzer are you affiliated with brid?

**Review by powerivq — APPROVED** [review]

File: extensions/amp-brid-player/0.1/test/test-amp-brid-player.js [hunk 1]
```diff
@@ -34,6 +34,18 @@ describes.realWin(
         bc.setAttribute('layout', 'responsive');
       }
 
+      // create config element if provided
+      if (config) {
+        const configElement = doc.createElement('script');
+        configElement.setAttribute('type', 'application/json');
+        if (typeof config == 'string') {
+          configElement.textContent = config;
```
Comment (erwinmombay) [inline extensions/amp-brid-player/0.1/test/test-amp-brid-player.js:42]: [eozmen410](https://github.com/eozmen410) do we need to make this assignment trusted types compatible?

**Comment by grajzer** [comment]
> 

Yes, I work in Brid and I made all previous changes to this component.

**Comment by grajzer** [comment]
Hi @erwinmombay please let me know if anything else is needed in regards to this, or when can we expect this change to go live.

**Comment by erwinmombay** [comment]
@grajzer apologies for late response.could you rebase and push the code out again. trying to get this green to merge

**Comment by grajzer** [comment]
Hi @erwinmombay, everything is updated now.

**Pull request merged**